.github/workflows/rspec_rubocop.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           libyaml-dev \
           libgmp-dev
     - name: Set up Ruby
-      uses: ruby/setup-ruby@ee2113536afb7f793eed4ce60e8d3b26db912da4 # v1.127.0
+      uses: ruby/setup-ruby@v1.143.0
```

Comment:
The version `v1.127.0` was not able to install Ruby 3.0 and 3.1 for some reason – it was failing with this error:

```
Error: Error: The process '/__t/Ruby/3.1.3/x64/bin/ruby' failed with exit code 127
```

Updating the version fixes the issue.